omero/institution/getting-started.html

```diff
@@ -9,8 +9,109 @@
+                <h5>OMERO manager person(s)</h5>
+                <ul class="dev-ops-links">
+                    This person is a go-between the users of OMERO and the server administrator. Very often this might be a microscopy facility manager.
+                    Typical tasks:
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html" target="_blank">Basic walkthrough</a></li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/group-user-management.html" target="_blank">Management of Groups/Users in OMERO</a></li>
+                    <li>Collating feedback from users</li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html#get-users-started" target="_blank">Introducing new users to OMERO</a></li>
+                    <li>Liaise with server administrator for features which require server configuration changes or installation of <a href="{{ site.baseurl }}/omero/apps/" target="_blank">OMERO Apps</a> and extensions.</li>
+                    <li>According to local needs, design and upload of new OMERO.scripts, possibly in cooperation with image analyst.</li>
+                    <li>Design and with server administrator implement an <a href="https://docs.openmicroscopy.org/omero/5.6.3/sysadmins/import-scenarios.html" target="_blank">import pipeline </a>, selecting the best possible <a href="{{ site.baseurl }}/omero/features/import/" target="_blank"> import setup</a> for the institution.</li>
+                </ul>
+            </div>
+        </div>
+        <!-- end What is needed -->
+        <a id="contacts"></a>
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>Key OMERO contacts</h3>
+        </div>
+        <hr>
+        <div class="row">
+                <h5>Contact OME Team</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="https://forum.image.sc/tag/omero" target="_blank">Main contact to OME Team is via image.sc, tag "omero"</a></li>
+                    <li><a href="{{ site.baseurl }}/support/" target="_blank">Check other means of support on this webpage</a></li>
+                </ul>
+                <h5>Regional groups and examples of institutions running OMERO</h5>
+                <ul class="dev-ops-links">
+                    Get in touch with your regional group to exchange experiences about OMERO from user perspective, contact or get inspired by OMERO setup in some institutions listed below.
+                    <li><a href="https://www.gerbi-gmb.de/WG6/RDM4mic" target="_blank">German BioImaging</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/server-setup-examples.html" target="_blank">Selected institutional OMERO deployments</a></li>
+                </ul>
+            </hr>
+        </div>
+        <!-- end Useful read, watch and contact -->
```

Comment:
The "Guides" section below probably needs it's own header, otherwise it looks like it's under the "Key OMERO contacts" section.

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/955aff51e8480dc3d5a6aa611a1fdf7c7048a6d6